nextflow_schema.json

```diff
@@ -57,6 +60,27 @@
             },
             "help_text": ""
         },
+        "bgc_antismash": {
+            "title": "BGC_antiSMASH",
+            "type": "object",
+            "description": "",
+            "default": "",
+            "properties": {
+                "bgc_skip_antismash": {
+                    "type": "boolean"
+                },
+                "bgc_antismash_databases": {
+                    "type": "string",
+                    "description": "Path to user-defined antiSMASH database",
+                    "default": "None"
+                },
+                "bgc_antismash_directory": {
+                    "type": "string",
+                    "description": "Path to user-defined antiSMASH directory",
```

Comment:
As this is critically important, it might be good to say this is required.

In fact... what was the realtionship again? You still need to supply this even if you allow the pipeline to download the database, right?

Reply:
Or could you just not reuse the same path actaully?